tests/UserRepositoryTest.php

```diff
@@ -147,4 +133,19 @@ protected function putJSONFixture($data, $fixtureName)
     {
         return file_put_contents($fixtureName, json_encode($data, JSON_PRETTY_PRINT));
     }
-}
+
+    protected function assertExceptionThrowed($class, $callback, $message)
+    {
+        try {
+            $callback();
+        } catch (Exception $error) {
+            if ($error instanceof $class){
+                return $this->assertEquals($error->getMessage(), $message);
+            }
+        }
+
+        echo 'fail';
+
+        return false;
```

Comment:
let's implement new AssertionExceptionExpectException class and throw it here

```suggestion
        throw new AssertionExceptionExpectException($expectedExceptionClass);
```